requirements.in

```diff
@@ -1,6 +1,6 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-ops
+ops>2
```

Comment:
Is >2 needed here?  I don't see anything we're changing that forces `>2`

If we did not have `>2` here, wouldn't we still get the same version of ops?